examples/postgres/verify.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e
+
+export NAME=postgres
+export DELAY=10
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+_psql () {
+    local postgres_client
+    postgres_client=(docker run -i --rm --network envoymesh -e "PGSSLMODE=disable" postgres:latest psql -U postgres -h envoy -p 1999)
+    "${postgres_client[@]}" "${@}"
+}
+
+run_log "Create a postgres database"
+_psql -c 'CREATE DATABASE test;'
```

Comment:
could we rename the db to maybe `testdb` - i was a bit confused at first by the name

Reply:
Make total sense... I also renamed the table name from `test` to `tbl`.